src/main/js/LoginPage.js

```diff
@@ -88,16 +76,6 @@ class LoginPage extends Component {
                                     </div>
                                 </div>
                             </div>
-                            <div>
-                                <ResetPasswordModal
```

Comment:
If the ResetPasswordModal is no longer being used here, should this components file be removed as well?

Reply:
I want to leave the backend parts and the Component in case we support more than one database user in the future and we need to bring this feature back.